### PR TITLE
Add verifier verifier fields

### PR DESCRIPTION
**Pull request by no2chem**
# Description

This PR adds the beneficiary id to the handshake message, which is necessary to distinguish between different verifiers.

### CODE REVIEW

**Comment by amytai** [comment]
Why is this necessary? i.e. any verifier can distinguish between other verifiers by things like IP address, etc. Why do they need to know the beneficiary address of a particular verifier?

**Comment by no2chem** [comment]
@amytai The IP address doesn't help you (which interface do I advertise(?)) What if you're behind a NAT? It's easier to just have some unique id that's advertised. This is especially for filtering out gossip: 

You could have verifier A advertised as 10.6.1.120 (private address) to verifier B, and 121.3.5.3 (public address) to verifier C.

**Comment by amytai** [comment]
I understand that verifiers can be behind NATs, but I forgot that verifiers need a globally unique ID to reduce the outgoing messages as well.

File: src/verifier.proto [hunk 1]
```diff
@@ -32,6 +32,8 @@ message VerifierVerifierHandshakeMessage {
   string protocol_version = 1;
   // The verifier version
   string version = 2;
+  // The beneficiary ID, 32 bytes BE
```
Comment (amytai) [inline src/verifier.proto]: Can you please expand to `big endian`, instead of BE? 

otherwise, LGTM

**Review by amytai — APPROVED** [review]

**Pull request merged**